static/app/views/issues/navigation.tsx

```diff
@@ -94,18 +96,44 @@ export function IssueNavigation({children}: IssuesWrapperProps) {
               baseUrl={baseUrl}
             />
           )}
-          <SecondaryNav.Section title={t('Configure')}>
-            <SecondaryNav.Item
-              to={`${baseUrl}/alerts/rules/`}
-              activeTo={`${baseUrl}/alerts/`}
-              analyticsItemName="issues_alerts"
-            >
-              {t('Alerts')}
-            </SecondaryNav.Item>
-          </SecondaryNav.Section>
+          <ConfigureSection baseUrl={baseUrl} />
```

Comment:
Should this be conditional? ex. show Alerts if they don't have the feature flag, show Monitors and Automations if they do?

Reply:
If you check the component, it is conditional! Component composition is a nice way to clean up branching logic like that

Reply:
Oh I totally missed that 🫣